api/context/src/main/java/io/opentelemetry/context/StrictContextStorage.java

```diff
@@ -65,7 +65,7 @@
  * > }
  * }</pre>
  */
-final class StrictContextStorage implements ContextStorage {
+final class StrictContextStorage implements ContextStorage, AutoCloseable {
```

Comment:
could you add a bit of javadoc to this class that explains that the close implementation does. I see it on the method, but I think it would be additionally helpful to have it at the class level.  (Also, I think that the example is using out of date APIs while you're in here. There is no `ContextStorage.set(...)` any more)

Reply:
Sure, thanks